test/org/sagebionetworks/bridge/play/controllers/AuthenticationControllerTest.java

```diff
@@ -172,10 +180,13 @@ public void testCode() throws Exception {
                 }
             }
         });
+        } catch(Exception e) {
+            e.printStackTrace();
+        }
```

Comment:
Removed one of the try blocks left over from testing, and rethread the inner catch.